nabcommon/hardware.py

```diff
@@ -12,3 +13,17 @@ def device_model():
     except (FileNotFoundError):
         model = platform.platform()
     return model
+
+
+def is_pi_zero(model):
+    """
+    Return True if given device model is a Raspberry Pi Zero / Zero 2,
+    False otherwise
+    """
+    matchObj = re.match(r"Raspberry Pi Zero", model)
+    if matchObj:
```

Comment:
`re` seems unnecessary here. Could you please use this simpler construct instead:
```
if "Raspberry Pi Zero" in model:
```

Reply:
Indeed. Will do.